src/Components/Topbar/Topbar.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import Toolbar from '@material-ui/core/Toolbar';
 import AppBar from '@material-ui/core/AppBar';
-import Button from '@material-ui/core/Button';
 import IconButton from '@material-ui/core/IconButton';
+import Badge from '@material-ui/core/Badge';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import MailIcon from '@material-ui/icons/Mail';
 import MenuIcon from '@material-ui/icons/Menu';
+import AccountCircle from '@material-ui/icons/AccountCircle';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
-
-const drawerWidth = 240;
+import Typography from '@material-ui/core/Typography';
+import configs from '../../settings';
 
 const useStyles = makeStyles((theme) => ({
     appBar: {
```

Comment:
We need to decide what pattern use. In Button component we have a _styles.js file and inside the Button.js we import the styles and use them as parameter to makeStyles function. I think that way is better, separating style object from Component JS file.